src/System.IO.FileSystem/tests/Directory/Delete.cs

```diff
@@ -202,12 +202,15 @@ public void UnixShouldBeAbleToDeleteHiddenDirectory()
             Assert.False(Directory.Exists(testDir));
         }
 
-        [ConditionalFact(typeof(PlatformDetection), "IsNotRedHat69")]
+        [Fact]
         [OuterLoop("Needs sudo access")]
         [PlatformSpecific(TestPlatforms.Linux)]
         [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
         public void Unix_NotFoundDirectory_ReadOnlyVolume()
         {
+            if (PlatformDetection.IsRedHat69)
+                return; // [ActiveIssue(https://github.com/dotnet/corefx/issues/21920)]
```

Comment:
Why the switch away from using ConditionalFact?

Reply:
ConditionalFact implies permanent. This is an active issue. (But I can't use ActiveIssue with PlatformDetection)

Reply:
> ConditionalFact implies permanent. This is an active issue. (But I can't use ActiveIssue with PlatformDetection)

FWIW, I don't believe [ConditionalFact] implies permanence any more or less than code in the body of the method does.  In either case, the intent is clarified by adding a comment as you've done, e.g.
```C#
[ConditionalFact(typeof(PlatformDetection), "IsNotRedHat69")] // [ActiveIssue(https://github.com/dotnet/corefx/issues/21920)]
```

Reply:
You're right I'll do that next time. I do want to have the attribute in the comment for future grepping.